### ISSUE TITLE
Minor updates on eslint-airbnb-config keep breaking my tests

### ISSUE THREAD

**Issue by olalonde** [airbnb/javascript#660]
It seems every once in a while eslint-airbnb-config introduces changes that break my lint tests. Most recently: 

```
    9:13  error  There should be no space after '['            array-bracket-spacing
    9:51  error  There should be no space before ']'           array-bracket-spacing
   10:5   error  Unquoted reserved word `default` used as key  quote-props
   16:5   error  Unquoted reserved word `default` used as key  quote-props
   21:5   error  Unquoted reserved word `default` used as key  quote-props
   22:13  error  There should be no space after '['            array-bracket-spacing
   22:36  error  There should be no space before ']'           array-bracket-spacing
   26:5   error  Unquoted reserved word `default` used as key  quote-props
   27:13  error  There should be no space after '['            array-bracket-spacing
   27:26  error  There should be no space before ']'           array-bracket-spacing
   31:5   error  Unquoted reserved word `default` used as key  quote-props
```

Those errors were not present in 3.0.0 but started showing up after I updated to 3.2.3. Could maintainers bump the major version of the npm package when such breaking changes are introduced?

**Comment by ljharb** [comment]
@olalonde 3.0.0 was just released 2 days ago, and 3.0.1 an hour ago. Did you mean v3.0.1?

The reality of a linter config is that almost every change could be "breaking" by typical semver standards. If I strictly followed what semver conventionally implies, there would only be patch (loosening enforcement of something, which almost never happens) or major (tightening enforcement of something, the primary purpose of a linter config).

The rubric I'm applying here is: if a linter rule is simply enabling enforcement of something our styleguide already recommends, it's minor. If it's dropping enforcement because our styleguide doesn't require it, it's patch. The only time it's a major bump is if the eslint config is changing as a result of an actual styleguide change.

**Comment by olalonde** [comment]
Ok thanks for the clarification. I'm not sure where I got that version number :/
